spec/models/post_mover_spec.rb

```diff
@@ -1,6 +1,14 @@
 require 'rails_helper'
 
 describe PostMover do
+  RSpec::Matchers.define :a_moderator_post do |attributes|
```

Comment:
Hmm I'm not really a fan of ths matcher here. I prefer to have less custom rspec magic and just be explicit in our tests.